components/esp_lvgl_port/Kconfig

```
@@ -0,0 +1,7 @@
+menu "ESP LVGL PORT"
```

```suggestion
menu
```
Could we make the menu hidden like this? (I have not tried it, I'd just like to avoid an empty menu in menuconfig)

---

This is not working, there must be a string